migtests/tests/pg/omnibus/expected_files/expectedAssessmentReport.json

```diff
@@ -477,21 +477,21 @@
 		{
 			"FeatureName": "GIST indexes",
 			"Objects": [
-				{
```

Comment:
why remove?

Reply:
this was a false positive becasue of earlier approach of regex where we just check for `gist` keyword at the end and this index has custom method as `gist2` which is not problematic if user has it target.

Reply:
Oh interesting. How do you even create a custom index method gist2!?

Reply:
Are we sure that it will be possible to create the same on YB?

Reply:
> How do you even create a custom index method gist2!?

Using `CREATE ACCESS METHOD` we can define custom index handler functions refer - https://www.postgresql.org/docs/17/sql-create-access-method.html

>Are we sure that it will be possible to create the same on YB?


It is not supported as of now https://github.com/yugabyte/yugabyte-db/issues/10693, but we can't detect it as such as don't export CREATE ACCESS METHOD DDLs .